csrc/executor_kernel_arg.h

```diff
@@ -65,6 +65,14 @@ class KernelArgumentHolder {
     return arguments_.at(ind).get();
   };
 
+  auto begin() const {
+    return arguments_.begin();
+  }
+
+  auto end() const {
+    return arguments_.end();
+  }
```

Comment:
This is to allow using range-based for-loops on a non-const KernelArgumentHolder.

Reply:
Separated to https://github.com/NVIDIA/Fuser/pull/2780.